pokemongo_bot/__init__.py

```diff
@@ -34,10 +34,15 @@ def start(self):
 
     def take_step(self):
         location = self.navigator.take_step()
-        cells = self.find_close_cells(*location[0:2])
+        self.work_cells(1, self.position[0:2])
 
-        for cell in cells:
-            self.work_on_cell(cell, location)
+  def work_cells(self, work_on_forts=1, location=None): 
+        if (location is None): 
+            location = self.position[0:2] 
```

Comment:
Please, don't get location from `self.position`, get if from `(self.api._position_lag, self.api._position_lng)`. Usage of `self.position` is being slowly eliminated from the project.